.gitignore

```diff
@@ -20,9 +20,9 @@ equiFLOW/
 java/
 python/
 nMHDust_progs/
-utilities/
```

Comment:
Add this back.  Use `git add -f filename` to force add something in a hidden directory.